BrainPortal/app/models/cbrain_task.rb

```diff
@@ -580,6 +580,7 @@ def invoke_after_status_transition_callbacks(from_state, to_state) #:nodoc:
   # This method compares a params hash table +old_params+ with
   # a +new_params+ hash provided, and log all the
   # differences. The task object itself is not changed.
+  # The method is overriden in BoutiquesTask subclass
```

Comment:
Normally when a method is being described, we never need to mention where the method is overridden in subclasses because that's kind of a dependency to goes the wrong way. If BoutiquesTask disappear in refactoring, your message is now nonsensical.

Reply:
Nonsensical, yet harmless. But you are right; it seems to be a general accepted practice to avoid mentioning the child in the parent method comments. Even though I never had explicit codified rules against that, I prefer to bow to public opinion.